api/cache_test.go

```diff
@@ -86,13 +86,15 @@ func (suite *CacheHandlerTestSuite) TestClearCache() {
 	group.SetTestID("/dir")
 	group.On("List", mock.Anything).Return([]plugin.Entry{}, nil)
 
+	reqCtx := context.WithValue(context.Background(), mountpointKey, "/")
```

Comment:
This could use some tests around the notCachedPathResponse error.

Reply:
Add a TODO?